antfs-cli.py

```diff
@@ -337,23 +326,22 @@ def main():
         g = AntFSCLI(options.upload)
         try:
             g.start()
-        except:
+        finally:
             g.stop()
-            raise
     except Device.ProfileVersionException as e:
-        print "\nError:", str(e), "\n\nThis means that", \
-                AntFSCLI.PRODUCT_NAME, "found that your data directory " \
-                "stucture was too old or too new. The best option is " \
-                "probably to let", AntFSCLI.PRODUCT_NAME, "recreate your " \
-                "folder by deleting your data folder, after backing it up, " \
-                "and let all your files be redownloaded from your sports " \
-                "watch."
+        print("\nError:%s\n\nThis means that % found that your data directory ",
```

Comment:
Missing an `s` here in `%s`? Also, there is a leftover comma that probably doesn't belong here.